boot/boot.go

```diff
@@ -30,8 +30,7 @@ import (
 	"github.com/snapcore/snapd/snap"
 )
 
-// A BootParticipant is a snap that is involved in a device's boot
-// process.
+// A BootParticipant is a snap that is involved in a device's boot process.
```

Comment:
// A BootParticipant handles the boot process details for a snap involved in it. ?